src/Utilities/TimerManager.h

```diff
@@ -154,16 +152,28 @@ struct TimerIDName_t
 template<class T>
 using TimerNameList_t = std::vector<TimerIDName_t<T>>;
 
-template<class T, class TIMER>
-void setup_timers(std::vector<std::reference_wrapper<TIMER>>& timers,
-                  TimerNameList_t<T> timer_list,
-                  timer_levels timer_level     = timer_level_fine,
-                  TimerManager<TIMER>* manager = &timer_manager)
+template<class TIMER>
+class TimerList : public std::vector<std::reference_wrapper<TIMER>>
 {
-  timers.reserve(timer_list.size());
-  for (int i = 0; i < timer_list.size(); i++)
-    timers.push_back(*manager->createTimer(timer_list[i].name, timer_level));
-}
+public:
+  TimerList() = default;
```

Comment:
Is this default constructor used somewhere?

Reply:
I removed all but 2, which are using the old type.